terraform/modules/vpc/main.tf

```
@@ -0,0 +1,69 @@
+data "aws_availability_zones" "available" {}
```

"available" is probably the default state but making it explicit is easier to read. I was also confused by the variable name because the docs say `state`:
> "Can be either "available", "information", "impaired" or "unavailable".

```suggestion
data "aws_availability_zones" "azs" {
    state = "available"
}
```

---

Yeah, it should have `state = "available"`. But then the variable name `available` should be fine?

---

Sure